lms/djangoapps/mobile_api/testutils.py

```diff
@@ -123,8 +146,228 @@ def test_other_user(self):
         self.api_response(expected_response_code=404, username=other.username)
 
 
+class MobileAPIMilestonesMixin(object):
+    """
+    Tests the Mobile API decorators for milestones.
+
+    The two milestones supported in these tests are entrance exams and
+    pre-requisite courses. If either of these milestones are unfulfilled,
+    the mobile api will appropriately block content until the milestone is
+    fulfilled.
+    """
+    MILESTONE_ERROR = {'developer_message': 'Cannot access content with unfulfilled pre-requisites or unpassed entrance exam.'}  # pylint: disable=line-too-long
+
+    # Enrollment list hides enrolled courses, milestone courses are not hidden
+    ALLOW_ACCESS_TO_MILESTONE_COURSE = False  # pylint: disable=invalid-name
+
+    def __init__(self):
+        self.entrance_exam = None
+        self.problem_1 = None
+        self.milestone = None
+        self.milestone_relationship_types = None
+        self.request = None
```

Comment:
My preference is to quiet this error.  This is a case of pylint being too stringent; tradeoff between concise/precise code and (needlessly) initializing variables in a dynamically typed language.